src/ec/packages/ec-component-checkbox/checkbox-group.html.twig

```diff
@@ -0,0 +1,103 @@
+{% spaceless %}
+
+{#
+  Parameters:
+  - "label_id" (string) (default: '')
+  - "items" (array) (default: []): array of checkbox_button
+  - "name" (string) (default: '')
+  - "helper_id" (string) (default: '')
+  - "invalid" (boolean) (default: false)
+  - "required" (boolean) (default: false)
+  - "extra_classes" (optional) (string) (default: '')
+  - "extra_attributes" (optional) (array) (default: [])
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+  Blocks:
+  - "label"
+  - "helper_text"
+  - "invalid_text"
+  - "optional_text"
+  - "required_text"
+#}
+
+{# Internal properties #}
+
+{% set _label_id = label_id|default('checkbox-default-label') %}
+{% set _items = items|default([]) %}
+{% set _name = name|default('') %}
+{% set _helper_id = helper_id|default('') %}
+{% set _invalid = invalid|default(false) %}
+{% set _required = required|default(false) %}
+
+{% set _helper_text = helper_text|default('') %}
+{% set _invalid_text = invalid_text|default('') %}
+{% set _required_text = required_text|default('*') %}
+{% set _optional_text = optional_text|default('') %}
+{% set _label = label|default('') %}
+
+{% set _css_class = 'ecl-form-group ecl-form-group--checkbox' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<fieldset
+  class="{{ _css_class }}"
+  aria-labelledby="{{ _label_id }}"
```

Comment:
i don't see any aria-labelledby attribute in the checkbox fieldset on ECL.

Reply:
done